js/private/translate_pnpm_lock.bzl

```diff
@@ -148,33 +147,22 @@ _ATTRS = {
         doc = """The basename for the node toolchain repository from @build_bazel_rules_nodejs.""",
         default = "nodejs",
     ),
-    "yq_repository": attr.string(
-        doc = """The basename for the yq toolchain repository from @aspect_bazel_lib.""",
-        default = "yq",
+    "yq": attr.string(
+        doc = """The label to the yq binary to use. If executing on a windows host, the .exe extension will be appended if missing.""",
```

Comment:
lets also check for `.cmd` and `.bat` which are also valid binary extensions on Windows

Reply:
You mean check for the existence of `.exe`, then `.bat` if it doesn't exist, and so forth?

Reply:
Yeah, if windows and extension is already `.exe`, `.bat` or `.cmd` then don't append `.exe`

Reply:
Oh, I thought you meant run several `rctx.execute`s to find out which one actually exists.